src/EnergyPlus/HeatBalanceMovableInsulation.cc

```diff
@@ -140,7 +140,8 @@ namespace HeatBalanceMovableInsulation {
 		// (I)BLAST legacy routine OMVINS
 
 		// USE STATEMENTS:
-		// na
+		using DataHeatBalance::WindowGlass;
```

Comment:
In general, I'd say just go ahead and write `DataHeatBalance::WindowGlass` in the below code instead of a using statement.  For this small of a function, it's not a big deal either way, but with larger functions, if there are many using statements, it gets hard to understand where each variable is coming from.